pkgcfg/pkgcfg.go

```diff
@@ -55,6 +55,7 @@ func CreateFiles(pkg, rootDir string) error {
 		}
 		var buf bytes.Buffer
 		if err = t.Execute(&buf, map[string]string{
+			"UserRoot":             rootDir,
```

Comment:
Is root dir the absolute path to project directory?

Reply:
yep. I named it UserRoot because we also have DevboxRoot which is the path to `.devbox`

Reply:
we also have the concept of `configDir` which is the path to where the project's `devbox.json` is kept. 

ugh, although this `configDir` is written as `Devbox.srcDir` in `devbox.go`. I can send a PR to update that.